community/deb.sh

```diff
@@ -79,10 +79,19 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+add_pgdg ()
+{
+  echo "Registering PostgreSQL PGDG dependencies..."
+  echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+  apt-get install ca-certificates
+  curl -L https://www.postgresql.org/media/keys/ACCC4CF8.asc 2> /dev/null | apt-key add - &>/dev/null
+}
+
 main ()
 {
   detect_os
   curl_check
+  add_pgdg
```

Comment:
Here are the three scenarios I envision, as seen by the user:

# Debian

## PostgreSQL 9.5 Found

```sh
Checking for postgresql-9.5...
Detected postgresql-9.5...
# no further PGDG-related output
```

## PostgreSQL 9.5 Not Found, `pgdg.list` File Absent

```sh
Checking for postgresql-9.5...
Installing PostgreSQL Apt Repository (PGDG)...
# no further PGDG-related output
```

## PostgreSQL 9.5 Not Found, `pgdg.list` File Present

```sh
Checking for postgresql-9.5...
Unable to install PostgreSQL Apt Repository

The file /etc/apt/sources.list.d/pgdg.list already exists.

Contact engage@citusdata.com with information about your system for help.
# exit with status 1
```

# Red Hat

## PostgreSQL 9.5 Found

```sh
Checking for postgresql95-server...
Detected postgresql95-server...
# no further PGDG-related output
```

## PostgreSQL 9.5 Not Found

```sh
Checking for postgresql95-server...
Installing PostgreSQL Yum Repository (PGDG)...
# no further PGDG-related output
```